### ISSUE TITLE
How is the new 2.4.1 config file path working?

### ISSUE THREAD

**Issue by NicolasMassart** [vale-cli/vale#242]
I updated my CircleCI workflow from Vale 2.4.0 to the new 2.4.1 version. 

I use the following CI steps to install and run Vale in a Docker container:

```
    steps:
      - checkout
      - run: git submodule sync
      - run: git submodule update --init
      - run:
          name: Install dependencies
          command: |
            curl -sfL https://install.goreleaser.com/github.com/ValeLint/vale.sh | sh -s v2.4.1
      - run:
          name: Run Vale
          command: |
            ./bin/vale --config ./common/build_tools/vale/.vale.ini --glob='*.{md}' . | tee ./vale.out
```

Now the [config file](https://docs.errata.ai/vale/cli#--config) is not found and I have the following message:

```
E200 Unable to locate a configuration file.
See https://docs.errata.ai/vale/config#search-process for more information.
```

I did not see any info about changing the path style for the config.

When I revert back to 2.4.0 it works again.
Thanks for your help!

**Comment by jdkato** [comment]
Yes, this was a regression related to the new error-handling changes (#238). It should be fixed in v2.4.2.

**Comment by NicolasMassart** [comment]
I confirm it works fine now. Thanks!
